### ISSUE TITLE
Use own StackStorm "init" Docker images in Test-Kitchen

### ISSUE THREAD

**Issue by arm4b** [StackStorm/chef-stackstorm#30]
Those images already have everything pre-installed to run properly default `init` system for specific OS.

See:
https://github.com/StackStorm/ansible-st2/blob/master/.kitchen.yml#L25-L39

**Comment by shortdudey123** [comment]
Generally speaking its best to test with non-baked images for kitchen

**Comment by arm4b** [comment]
@shortdudey123 You mean it's better to leave it as is?
Eg. configure everything at a run-time.

**Comment by shortdudey123** [comment]
Yes, IMO that avoids caveats that are avoided w/ pre-baked images (since the purpose of them is to shortcut and speed things up)

**Comment by jvrplmlmn** [comment]
> Yes, IMO that avoids caveats that are avoided w/ pre-baked images (since the purpose of them is to shortcut and speed things up)

Agree on this.

---

@armab where can we find the `Dockerfile`s for `stackstorm/packagingtest`?
